proposals/lip-0049.md

```diff
@@ -573,8 +573,6 @@ def execute(ccu: Transaction, ccm: CCM) -> None:
     )
 ```
 
-The `terminateChain` function is defined in [LIP 0045][lip-0045].
```

Comment:
Why should it be removed from there?

Reply:
In general, we are quite inconsistent with citing LIP45. I think we can just remove all references since we do it in the beginning of the specs

Reply:
This should be removed because it is already linked earlier in this LIP. If you want to keep it, link it consistently (i.e., use [lip-0045#createTerminatedStateAccount] in both links).

Reply:
done